src/components/AssetsPage/index.jsx

```diff
@@ -10,43 +10,98 @@ import { getAssets } from '../../data/actions/assets';
 import edxBootstrap from '../../SFE.scss';
 import styles from './AssetsPage.scss';
 
-class AssetsPage extends React.Component {
+export class AssetsPage extends React.Component {
```

Comment:
I've exported the unconnected component to be able to test it independent of the Redux store.